modules/migration-understanding-resources-mtc-migrates.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * migration/migrating_3_4/migrating-application-workloads-3-4.adoc
+// * migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
+
+[id='about-resources-mtc-migrates_{context}']
+= About the resources migrated
+[role="_abstract"]
+
+You can use the {mtc-full} ({mtc-short}) to migrate the following types of resources:
+
+* Namespaces specified in migration plans. Migration plans have the granularity of a namespece.
+
+* Namespace-scoped resources. When a namespace is selected for migration, then all the objects and resources (services, pods, etc.) pertaining to that namespace are also selected for migration. Additionally, if a resource scoped by a namespace depends on a resource that is scoped by a cluster, then the resource scoped by the cluster is also migrated by the  {mtc-full} Operator.
++
+For instance, a Security Context Constraint (SCC) is a resource scoped by a cluster and a service account (SA) is a resource scoped by a namespece. If an SA exists in a namespace that you select for migration, then the {mtc-full} Operator automatically locates any SCCs that are applied to this SA and include those SCCs in the migration as well. Similarly, the {mtc-full} Operator migrates Persistent Volume Claims linked to the Persistent Volumes of the namespace.
+
+* Custom Resources (CRs) and Custom Resource Definitions (CRDs). Any CRs that are scoped by a namespace are automatically included in the migration of the namespace. Therefore, the {mtc-full} Operator also migrates the CRDs associated with those CRs.
```

Comment:
Remove "Any" and "Therefore". There is no cause-and-effect relationship. Remove "also".
No Operator. MTC performs the migration.